client/src/App.scss

```diff
@@ -304,10 +265,10 @@ Add User Form
   display: flex;
   flex-direction: column;
   padding: 1rem;
-}
 
-.user-add__label {
-  padding-bottom: 0.5rem;
+  fieldset {
+    margin-bottom: 0;
+  }
```

Comment:
instead of this rule, try using a class name and passing that class name to the input

```
<TextInput className="..."
```